src/main/webapp/src/components/sentimentAnalysisInfo.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { createMuiTheme, withStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Tooltip from '@material-ui/core/Tooltip';
+import InfoIcon from '@material-ui/icons/Info';
+import axios from "axios";
+
+const styles = (theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+  },
+  sentimentVariableInfo: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+  },
+  sentimentVariableInfoIcon: {
+    paddingLeft: "15px",
+  },
+});
+
+const LargeTextTooltip = withStyles({
+  tooltip: {
+    fontSize: "14px",
+  }
+})(Tooltip);
+
+/**
+ * Displays sentiment analysis info for the lyrics.
+ */
+class SentimentAnalysisInfo extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      sentimentAnalysisInfo: {},
+      isLoading: false,
+      error: null,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ isLoading: true });
+
+    axios
+      .post("/sentiment", {}, {
+        headers: {
+          'Content-Type': 'application/json',
+          'Access-Control-Allow-Origin': '*'
+        },
+        params: {
+          lyrics: this.props.lyrics  
+        }
+      })
+      .then((result) => result.data)
+      .then((sentimentAnalysisInfo) =>
+        this.setState({
+          sentimentAnalysisInfo: sentimentAnalysisInfo,
+          isLoading: false,
+        })
+      )
+      .catch((error) =>
+        this.setState({
+          error,
+          isLoading: false,
+        })
+      );
+  }
+
+  render() {
+    const classes = this.props.classes;
+
+    if (this.state.error) {
+      return (
+        <div>
+          <p>{this.state.error.message}</p>
+        </div>
+      );
+    }
+
+    if (this.state.isLoading) {
+      return (
+        <div>
+          <CircularProgress style={{ color: "black" }} />
```

Comment:
nit: can we centralize the style of the CircularProgress somewhere? We should be consistent across all the parts of the app, so there is no need to set `"color: black"` everywhere, because if we change our mind we need to replace it everywhere.

This change should not be part of this Pull Request, since it's unrelated.

Reply:
sure